src/picosvg/svg.py

```diff
@@ -1358,13 +1419,26 @@ def _do_not_inherit(*_):
 )
 
 
+def _drop_default_attrib(attrib: MutableMapping[str, Any]):
+    for attr_name in sorted(attrib.keys()):
+        value = attrib[attr_name]
+        default_value = attrib_default(attr_name, default=None)
+        if default_value is None:
+            continue
+        if isinstance(default_value, float):
+            value = float(value)
```

Comment:
i don't understand why you cast value to a float when isinstance check already says it is a float?